include/chopper/count/count_kmers.hpp

```diff
@@ -30,32 +30,24 @@ using sequence_file_type = seqan3::sequence_file_input<mytraits,
                                                        seqan3::type_list<seqan3::format_fasta, seqan3::format_fastq>>;
 
 template <typename seq_type, typename compute_view_type>
-void compute_hashes(seq_type && seq,
-                    compute_view_type && compute_fn,
-                    configuration const & config,
-                    robin_hood::unordered_node_set<uint64_t> & result,
-                    chopper::sketch::hyperloglog & sketch)
+void add_hashes_to_sketch(seq_type && seq, compute_view_type && compute_fn, chopper::sketch::hyperloglog & sketch)
```

Comment:
```suggestion
void hash_and_add_sequence_to_sketch(seq_type && seq, compute_view_type && compute_fn, chopper::sketch::hyperloglog & sketch)
```

Reply:
Actually, this might just be a lambda in `count_kmers`:

```cpp
auto hash_and_add_to_sketch = [&sequence_vector, &sketch] (auto && compute_fn)
{
    for (auto && seq : sequence_vector)
        for (auto && hash : seq | compute_fn)
            sketch.add(reinterpret_cast<char*>(&hash), sizeof(hash));
}
```

and then 

```cpp
        if (config.disable_minimizers)
            hash_and_add_to_sketch(compute_kmers);
        else
            hash_and_add_to_sketch(compute_minimiser);
```

\+- maybe some naming